server/apps/immich/src/api-v1/server-info/server-info.service.ts

```diff
@@ -6,7 +6,7 @@ import diskusage from 'diskusage';
 @Injectable()
 export class ServerInfoService {
   async getServerInfo(): Promise<ServerInfoResponseDto> {
-    const diskInfo = await diskusage.check(APP_UPLOAD_LOCATION);
+    const diskInfo = await diskusage.check(process.env.UPLOAD_DIR || APP_UPLOAD_LOCATION);
```

Comment:
I don't see the `UPLOAD_DIR` in the `.env.example`. Is this a left over from previous changes?

Reply:
Or is this for running the app as a standalone?

Reply:
I wanna hide this env for docker user now. It's really useful with dockerless usage by using pm2, systemd, openrc, ...

Reply:
Maybe there is a better way to implement this with documentation 🤔

Reply:
I just start my first step.  If there are many docker-less users later, document will be available. Or we can support directly through installation scripts https://github.com/immich-app/immich/pull/556, so user don't need document.

Reply:
I also think we need a documentation page to put such guides. I'm waiting for https://github.com/immich-app/immich/issues/244